src/components/Graph/Graph.tsx

```diff
@@ -91,6 +91,8 @@ const Graph: React.FC<GraphProps> = ({
     const generateNodes = (nodes: any[]): any => {
       const minMax = getMinMax(nodes);
       return nodes.map((n: any) => {
+      const imageURL=`image://https://avatars.githubusercontent.com/` + n.name;
+      if(n.name.includes('/')){
```

Comment:
Hey @smitsekhadiaa, it is a feasible way and brings minimal change to existing code.  You can add some comment on these lines (I mean in Graph.tsx not on GitHub) to explain why there is a if-branch.

Reply:
yes got it @tyn1998 
i'll add the comments to the code right now.👍